web-local/src/routes/(viz)/dashboard/[name]/+page.svelte

```diff
@@ -24,26 +25,45 @@
   $: ({ instanceId } = $runtime);
 
   $: filePaths = data.metricsView.meta?.filePaths as string[];
-  $: projectParserQuery = useProjectParser(queryClient, instanceId);
+  $: dashboard = useDashboard(instanceId, metricsViewName);
+  $: measures = $dashboard.data?.metricsView?.state?.validSpec?.measures ?? [];
+  $: projectParserQuery = useProjectParser(
+    queryClient,
+    instanceId,
+    measures.length > 0,
```

Comment:
I don't think we need to prevent the Project Parser request when there are no measures. Rather we should prevent it when the mock user is not an admin.